spring-xd-dirt/src/main/java/org/springframework/xd/dirt/rest/ModulesController.java

```diff
@@ -73,13 +73,12 @@ public ModulesController(ModuleDefinitionService moduleDefinitionService,
 	@RequestMapping(value = "", method = RequestMethod.GET)
 	@ResponseStatus(HttpStatus.OK)
 	@ResponseBody
-	public PagedResources<ModuleDefinitionResource> list(Pageable pageable,
-			PagedResourcesAssembler<ModuleDefinition> assembler,
-			@RequestParam(value = "type", required = false) ModuleType type) {
+	public PagedResources<? extends ModuleDefinitionResource> list(Pageable pageable,
```

Comment:
weird format change (at least it doesn't align with the formatting in other methods). will fix it while merging.